### PR TITLE
Purge old GUI-related code

### PR DESCRIPTION
**Pull request by zestyr**

### CODE REVIEW

File: lbryum/simple_config.py [hunk 1]
```diff
@@ -153,9 +153,6 @@ def save_user_config(self):
         f = open(path, "w")
         f.write(s)
         f.close()
-        if self.get('gui') != 'android':
```
Comment (zestyr) [inline lbryum/simple_config.py]: I'm unsure if this is supposed to be run anyway, kinda seems like it is

File: lbryum/simple_config.py [hunk 2]
```diff
@@ -153,9 +153,6 @@ def save_user_config(self):
...
-            import stat
-            os.chmod(path, stat.S_IREAD | stat.S_IWRITE)
```
Comment (zestyr) [inline lbryum/simple_config.py]: I'm unsure if this is supposed to be run anyway, kinda seems like it is

Comment (jackrobison) [inline lbryum/simple_config.py]: I think it's correct to remove it. The default config doesn't have a gui value, so it isn't being run in almost all cases currently.

**Comment by jackrobison** [comment]
Merged [here](https://github.com/lbryio/lbryum/commit/012403d8311eef3253140ed9813f46309943b87a)

**Pull request closed without merging**